src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/LocationAttribute.kt

```diff
@@ -0,0 +1,91 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.entity
+
+import jakarta.persistence.CascadeType
+import jakarta.persistence.Entity
+import jakarta.persistence.EnumType
+import jakarta.persistence.Enumerated
+import jakarta.persistence.FetchType
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.OneToMany
+import jakarta.persistence.OneToOne
+import jakarta.persistence.Table
+import org.hibernate.Hibernate
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+@Entity
+@Table(name = "location_attribute")
+data class LocationAttribute(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val locationAttributeId: Long = 0,
+
+  val locationKey: String,
```

Comment:
We shouldn't be storing the Location business key (assuming that is what this is????).

It should be the UUID of the Location.

Reply:
I agree - looking though various endpoints there are quite a few still using/returning the key - so as discussed, there is a wider sweep through required. However, I'll do this bit now as we know its the unique value.